README.md

```diff
@@ -116,6 +116,24 @@ All existing plugins follows a naming convention: `rnpm-plugin-<plugin name>` (e
 
 For further reading you can check our [example plugin](https://github.com/rnpm/rnpm-plugin-link)
 
+### Commands
+In the case you need an additional input from the user, you may make a `command` for this purpose. Commands works similar to the npm scripts.
+
+*Depdendency's `package.json`:*
+```json
+"rnpm": {
+  "commands": {
+    "prelink": "./bin/requestGAToken",
+    "postlink": "./bin/linkingSucceeded"
+  }
+}
+```
+In this scenario we're using custom `prelink` and `postlink` hooks for `rnpm-plugin-link` to tell `rnpm` that we want to run `prelink` script before and `postlink` after the linking process.
+
+**Note:** `commands` may be async and require some user input using third-party libraries (inquirer for instance). You don't need to worry about async queues, we do it for you under the rnpm hood.
```

Comment:
At least it would prevent stupid questions like:
- Ah? I've added it to scripts, but it doesnt work!
- Scripts section should be inside rnpm key
- O, wow! Thanks